modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -26,7 +26,7 @@ The `domain` value must be unique among all Ingress Controllers and cannot be up
 If empty, the default value is `ingress.config.openshift.io/cluster` `.spec.domain`.
 
 |`appsDomain`
-|`appsDomain` is an optional domain for AWS infrastructure to use instead of the one specified in the `domain` field when a Route is created without specifying an explicit host. If a value is entered for `appsDomain`, this value is used to generate default host values for the Route. Unlike `domain`, `appsDomain` can be modified after installation. You can use this parameter only if you set up a new Ingress Controller that uses a wildcard certificate.
+|`appsDomain` is an optional domain for cloud platform infrastructure to use instead of the one specified in the `domain` field when a Route is created without specifying an explicit host. If a value is entered for `appsDomain`, this value is used to generate default host values for the Route. Unlike `domain`, `appsDomain` can be modified after installation. You can use this parameter only if you set up a new Ingress Controller that uses a wildcard certificate.
```

Comment:
```suggestion
|`appsDomain` is an optional domain for {product-title} to use instead of the one specified in the `domain` field when a Route is created without specifying an explicit host. If a value is entered for `appsDomain`, this value is used to generate default host values for the Route. Unlike `domain`, `appsDomain` can be modified after installation. You can use this parameter only if you set up a new Ingress Controller that uses a wildcard certificate.
```